src/infra/ingest-datafusion/src/readers/shapefile.rs

```diff
@@ -98,7 +98,7 @@ impl ReaderEsriShapefile {
     }
 
     fn locate_shp_file(dir: &Path, subpath: Option<&str>) -> Result<PathBuf, ReadError> {
-        let is_shp_file = |p: &Path| -> bool { p.extension().map(|s| s == "shp").unwrap_or(false) };
+        let is_shp_file = |p: &Path| -> bool { p.extension().is_some_and(|s| s == "shp") };
```

Comment:
That's something I haven't seen ever yet.

Reply:
I agree, interesting fix